typescript/sdk/src/providers/SmartProvider/SmartProvider.ts

```diff
@@ -264,26 +264,40 @@ export class HyperlaneSmartProvider
         );
         const result = await Promise.race([resultPromise, timeoutPromise]);
 
+        const providerMetadata = {
+          providerIndex: pIndex,
+          rpcUrl: provider.getBaseUrl(),
+          method: `${method}(${JSON.stringify(params)})`,
+          chainId: this.network.chainId,
+        };
+
         if (result.status === ProviderStatus.Success) {
           return result.value;
         } else if (result.status === ProviderStatus.Timeout) {
           this.logger.debug(
-            `Slow response from provider #${pIndex}.${
-              !isLastProvider ? ' Triggering next provider.' : ''
-            }`,
+            `Slow response from provider:`,
+            { ...providerMetadata },
+            isLastProvider ? '' : 'Triggering next provider.',
```

Comment:
object should be the first arg, everything after the msg is interpolation values https://getpino.io/#/docs/api?id=loggerdebugmergingobject-message-interpolationvalues

Reply:
there's probably a good follow-up task to update all the other this.logger calls to do the same - i believe we didn't really update this after the switch to pino

Reply:
ugh i _really_ dislike this format. data followed by message makes very little sense to me

with that said, why do we care? the point is to be able to grep logs easily for, e.g., `Slow response from provider:` and not `Slow response from ${chain} provider ${index}:`

Reply:
so to further clarify– we are eliminating string interpolation hyperlane-side here, and what happens pino-side should not affect our ability to grep logs

Reply:
i made the requested update, and we can follow up on this as a chat offline